<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticesCellWidget</class>
 <widget class="QWidget" name="NoticesCellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>179</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="4">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelDate">
     <property name="text">
      <string>截止日期：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelFontSize">
     <property name="text">
      <string>字体大小(pt)：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QTextEdit" name="textEdit">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>在此输入公告内容</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>无限期</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textEdit</sender>
   <signal>textChanged()</signal>
   <receiver>NoticesCellWidget</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NoticesCellWidget</receiver>
   <slot>changeFontPtSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>labelDate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>NoticesCellWidget</receiver>
   <slot>onDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>NoticesCellWidget</receiver>
   <slot>onDateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChanged()</slot>
  <slot>changeFontPtSize(int)</slot>
  <slot>onDateChanged()</slot>
 </slots>
</ui>
